packages/react/src/contexts/treasure.tsx

```diff
@@ -300,6 +316,7 @@ const TreasureProviderInner = ({
         setRootElement: setEl,
         isUsingTreasureLauncher,
         openLauncherAccountModal,
+        trackCustomEvent,
```

Comment:
I feel like this could go in its own hook like `useTreasureAnalytics` instead of being part of the context since it doesn't affect the context state at all